app/views/groups/new.html.erb

```diff
@@ -2,20 +2,20 @@
 <% content_for :title, t("groups.new.title") %>
 <div class="container">
   <div class="row center-row">
-    <div class="col-xs col-sm col-md col-lg">
+    <div class="col col-sm col-md col-lg">
       <h2 class="submain-heading"><%= t("groups.new.main_heading") %></h2>
       <p class="main-description"><%= t("groups.new.main_description_html") %></p>
     </div>
   </div>
   <div class="row center-row">
-    <div class="col-xs-12 col-sm-12 col-md-7 col-lg-6">
+    <div class="col-12 col-sm-12 col-md-7 col-lg-6">
       <div class="container">
         <div class="center-row groups-background-card">
           <%= render "form", group: @group %>
         </div>
       </div>
     </div>
-    <div class="col-xs-12 col-sm-12 col-md-5 col-lg-6">
+    <div class="col-12 col-sm-12 col-md-5 col-lg-6">
       <%= image_tag "PNGs/newGroup.png", alt: "New Group Sketch", class: "groups-center-image center-image" %>
     </div>
   </div>
```

Comment:
These layout patterns are part of the site’s original design, and changing them could impact the overall look and feel. I think it would be best to get approval from the UI/UX team before making adjustments to ensure we maintain the intended visual consistency across the site.